### ISSUE TITLE
Draw only once for better exploration

### ISSUE THREAD

**Issue by EnricoLusaEGF** [treymo/joplin-link-graph#35]
The constant redrawing of the graph when opening a new note with the related elastic animation makes exploring the graph really uncomfortable in my opinion. Since there is already a "Redraw Graph" button, I think it would be nice to have a tick to avoid redrawing next to it. Clicking a topic would only recenter the graph to it. This way you can use the graph as a semi-persistent map to navigate your content. 

Later on, it would probably be a good idea to have the option to store the layout, so it will always present itself in the same way after closing the tab or the program.

**Comment by Ravichandra-C** [comment]
@treymo , Can I pick any issue from this repo and work to it? Also Could you please let me know if there are any issues that are beginner friendly?

PS: I'm sorry if this is not the right place to ask about this
